tests/test_core/test_alignment.py

```diff
@@ -2601,6 +2601,18 @@ def test_slice_align_info(self):
         self.assertTrue(len(sub_align) == 3)
         self.assertEqual(sub_align.info["key"], "value")
 
+    def test_to_str(self):
```

Comment:
this test not correct. It should be testing `aln._repr_html_()`. Probably easiest to check by counting the number of `<tr>` elements.